tauri/src/endpoints/asset.rs

```diff
@@ -76,7 +76,8 @@ pub fn load(
                   document.getElementsByTagName("head")[0].appendChild(css);
                 }})(`{css}`)
               "#,
-              css = asset_str
+              // Escape octal sequences, which aren't allowed in template literals
+              css = asset_str.replace("\\", "\\\\").as_str()
```

Comment:
what if there are already e.g. `\\\\`? Wouldn't that make `\\\\\\\\`? or is this a non-concern?

Reply:
That is desired behavior. Since the content is being injected into a template literal, the extra slashes will be reverted back to original when the content is actually injected into the dom.